globus_automate_client/cli/callbacks.py

```diff
@@ -1,12 +1,13 @@
 import json
 import os
 import pathlib
+import yaml
+import typer
```

Comment:
I'm not sure what triggered the move of the typer import. Ideally,we're running on a consistent config for isort and that's doing the job. Given that typer is alphabetically before yaml, I'm guessing isort wasn't run here.

@urimandujano Perhaps you can comment on isort config, running lint steps etc.

But, not a problem at all at this draft stage.

Reply:
vscode's isort integration isn't super well thought out. Since we don't have CI here it amounts to configuring vscode to run isort on save everytime OR running it manually. I added this in my vscodes `settings.json`:

`    "[python]": {
        "editor.codeActionsOnSave": {
            "source.organizeImports": true
        }
    },`